test/fail_compilation/traits.d

```diff
@@ -0,0 +1,24 @@
+/*
+REQUIRED_ARGS:
+PERMUTE_ARGS:
+*/
+
+/************************************************************/
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/traits.d(100): Error: getTargetInfo key "not_a_target_info" not supported by this implementation
+fail_compilation/traits.d(101): Error: string expected as argument of __traits getTargetInfo instead of 100
+fail_compilation/traits.d(102): Error: expected 1 arguments for getTargetInfo but had 2
+fail_compilation/traits.d(103): Error: expected 1 arguments for getTargetInfo but had 0
+fail_compilation/traits.d(104): Error: expected 0 arguments for allTargetInfos but had 1
```

Comment:
actual:
```
fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
fail_compilation/traits.d(104): Error: expected 0 arguments for `allTargetInfos` but had 1
```

Reply:
That's not what I get when I run locally...
I literally cut the pasted the error messages from the local execution :/

Reply:
Why is everything so bloody hard and time-wasting! >_<

Reply:
Ready for the windows tests to fail now...?